tests/acceptance/helpers.py

```diff
@@ -14,6 +22,17 @@
 
 PROJECT_ROOT = os.path.dirname(os.path.abspath(__file__))
 
+os.environ['OPENAPI_YAML'] = os.path.abspath('api/openapi-spec/openapi.yaml')
```

Comment:
You need to get file path from env, not the other way. 
assume that values is in os.environ['OPENAPI_YAML_FILEPATH']
and get value from this. Otherwsie, use default value.

Reply:
I did not understand what you mean, how can I assume that value exists in os.environ['OPENAPI_YAML_FILEPATH'] when i didnt set any value in it. Please elaborate this one